src/main/java/pl/xkoem/userinterface/language/LanguageName.java

```diff
@@ -0,0 +1,29 @@
+package pl.xkoem.userinterface.language;
+
+public enum LanguageName {
+    player_name_question("Select player name %symbol%"),
```

Comment:
Enum with small letters objects is weird

Reply:
done